lib/mongodb.js

```diff
@@ -751,6 +751,13 @@ function buildNearFilter(query, near) {
       },
     },
   };
+
+  var props = ['maxDistance', 'minDistance'];
+  props.forEach(function(p) {
+    if (near[p]) {
+      query.where[near.key]['near']['$' + p] = near[p];
```

Comment:
An issue I see with this logic is that it assumes maxDistance in defined in 'meter' units (mongo default).

Loopback allows for a unit property so the value should really be checked and converted to meter equiv. here.

Possible unit values to take into account:
       miles (default)
       radians
       kilometers
       meters
       miles
       feet
       degrees